inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/repository/DataProxyConfigRepository.java

```diff
@@ -56,11 +56,15 @@
 @Repository(value = "dataProxyConfigRepository")
 public class DataProxyConfigRepository implements IRepository {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(DataProxyConfigRepository.class);
+
+    private static final String KEY_SECOND_CLUSTER_TAG = "second_cluster_tag";
```

Comment:
Maybe using 'backup' is better than 'second'?

Reply:
Inlong group id change cluster tag from A to B. It is not a backup concept.
And second_cluster_tag have been used by online SortStandalone.

Reply:
OK.